compressible_flow/NICFD_nozzle/DataDriven/README.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Data-Driven Fluid Model
+permalink: /SU2_CFD/src/fluid/CDataDrivenFluid.cpp
+---
+# Using Data-Driven Fluid Models in SU2
+Modeling non-ideal fluid behavior can be difficult when limited to model equations or can be time consuming when performing large-scale simulations using the CoolProp fluid model. The data-driven fluid model class allows for the equation of state to be defined through a look-up table (LUT) or a set of multi-layer perceptrons (MLP), using an entropy-based equation of state. Given enough reference data and careful tuning of the initial conditions, it is possible to attain similar accuracies as the CoolProp library, while being compatible with the SU2 adjoint solver and a faster iteration time.  
+
```

Comment:
I added a modified version of the tutorial instructions to the website repo (feature_multilayer_peceptron). It explains how to use the data-driven fluid model and how to train a multi-layer perceptron to use as its input. However, in order to explain the entropy-based fluid model, I need to reference a paper which is yet to be published. This may take a couple of weeks. Do you think it's ok to merge this branch in the meantime or is it better to wait until the website tutorial instructions are complete?